[rospy.client][INFO] 2019-06-25 21:27:18,995: init_node, name[/joint_state_publisher], pid[10163]
[xmlrpc][INFO] 2019-06-25 21:27:18,995: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-25 21:27:18,996: Started XML-RPC server [http://dc5b429f784e:37531/]
[rospy.impl.masterslave][INFO] 2019-06-25 21:27:18,996: _ready: http://dc5b429f784e:37531/
[xmlrpc][INFO] 2019-06-25 21:27:18,997: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2019-06-25 21:27:18,997: ROS Slave URI: [http://dc5b429f784e:37531/]
[rospy.registration][INFO] 2019-06-25 21:27:18,998: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-25 21:27:19,099: registered with master
[rospy.rosout][INFO] 2019-06-25 21:27:19,099: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-25 21:27:19,101: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-25 21:27:19,102: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2019-06-25 21:27:19,305: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2019-06-25 21:27:19,371: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.core][INFO] 2019-06-25 21:28:04,246: signal_shutdown [signal-2]
[rospy.internal][INFO] 2019-06-25 21:28:04,249: topic[/joint_states] removing connection to /robot_state_publisher
[rospy.internal][INFO] 2019-06-25 21:28:04,249: topic[/rosout] removing connection to /rosout
[rospy.impl.masterslave][INFO] 2019-06-25 21:28:04,250: signal-2
[rospy.core][INFO] 2019-06-25 21:28:04,418: signal_shutdown [atexit]
[rospy.client][INFO] 2019-06-25 21:28:17,296: init_node, name[/joint_state_publisher], pid[10742]
[xmlrpc][INFO] 2019-06-25 21:28:17,297: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-25 21:28:17,297: Started XML-RPC server [http://dc5b429f784e:36193/]
[rospy.impl.masterslave][INFO] 2019-06-25 21:28:17,297: _ready: http://dc5b429f784e:36193/
[rospy.init][INFO] 2019-06-25 21:28:17,298: ROS Slave URI: [http://dc5b429f784e:36193/]
[xmlrpc][INFO] 2019-06-25 21:28:17,298: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-25 21:28:17,299: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-25 21:28:17,399: registered with master
[rospy.rosout][INFO] 2019-06-25 21:28:17,399: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-25 21:28:17,410: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-25 21:28:17,412: initializing /clock core topic
[rospy.simtime][INFO] 2019-06-25 21:28:17,415: connected to core topic /clock
[rospy.internal][INFO] 2019-06-25 21:28:17,653: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2019-06-25 21:28:17,750: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.internal][INFO] 2019-06-25 21:28:23,781: topic[/clock] adding connection to [http://dc5b429f784e:39015/], count 0
[rospy.internal][ERROR] 2019-06-25 21:51:49,169: Unable to initiate TCP/IP socket to dc5b429f784e:55135 (http://dc5b429f784e:39015/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 55135 to [172.17.0.2:34244 on socket 9]] [/joint_state_publisher].

[rospy.internal][ERROR] 2019-06-25 21:51:50,171: Unable to initiate TCP/IP socket to dc5b429f784e:55135 (http://dc5b429f784e:39015/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 55135 to [172.17.0.2:34322 on socket 16]] [/joint_state_publisher].

[rospy.internal][ERROR] 2019-06-25 21:51:52,174: Unable to initiate TCP/IP socket to dc5b429f784e:55135 (http://dc5b429f784e:39015/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: received a connection for a nonexistent topic [/clock] from [TCPROS connection on port 55135 to [172.17.0.2:34324 on socket 16]] [/joint_state_publisher].

[rospy.core][INFO] 2019-06-25 21:51:52,383: signal_shutdown [signal-2]
[rospy.internal][INFO] 2019-06-25 21:51:52,386: topic[/joint_states] removing connection to /robot_state_publisher
[rospy.internal][INFO] 2019-06-25 21:51:52,386: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2019-06-25 21:51:52,386: topic[/clock] removing connection to http://dc5b429f784e:39015/
[rospy.impl.masterslave][INFO] 2019-06-25 21:51:52,387: signal-2
[rospy.core][INFO] 2019-06-25 21:51:52,862: signal_shutdown [atexit]
[rospy.client][INFO] 2019-06-25 21:53:32,680: init_node, name[/joint_state_publisher], pid[16552]
[xmlrpc][INFO] 2019-06-25 21:53:32,680: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2019-06-25 21:53:32,680: Started XML-RPC server [http://dc5b429f784e:44137/]
[rospy.impl.masterslave][INFO] 2019-06-25 21:53:32,681: _ready: http://dc5b429f784e:44137/
[rospy.init][INFO] 2019-06-25 21:53:32,681: ROS Slave URI: [http://dc5b429f784e:44137/]
[xmlrpc][INFO] 2019-06-25 21:53:32,681: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2019-06-25 21:53:32,682: Registering with master node http://localhost:11311
[rospy.init][INFO] 2019-06-25 21:53:32,782: registered with master
[rospy.rosout][INFO] 2019-06-25 21:53:32,782: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-25 21:53:32,784: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-25 21:53:32,786: initializing /clock core topic
[rospy.simtime][INFO] 2019-06-25 21:53:32,788: connected to core topic /clock
[rospy.internal][INFO] 2019-06-25 21:53:33,036: topic[/joint_states] adding connection to [/robot_state_publisher], count 0
[rospy.internal][INFO] 2019-06-25 21:53:33,039: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2019-06-25 21:54:03,732: topic[/clock] adding connection to [http://dc5b429f784e:34901/], count 0
[rospy.core][INFO] 2019-06-25 21:57:52,060: signal_shutdown [signal-2]
[rospy.internal][INFO] 2019-06-25 21:57:52,069: topic[/joint_states] removing connection to /robot_state_publisher
[rospy.internal][INFO] 2019-06-25 21:57:52,070: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2019-06-25 21:57:52,070: topic[/clock] removing connection to http://dc5b429f784e:34901/
[rospy.impl.masterslave][INFO] 2019-06-25 21:57:52,071: signal-2
[rospy.internal][WARNING] 2019-06-25 21:57:52,071: Unknown error initiating TCP/IP socket to dc5b429f784e:43019 (http://dc5b429f784e:34901/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.core][INFO] 2019-06-25 21:57:52,536: signal_shutdown [atexit]
